src/oer/de.rs

```diff
@@ -0,0 +1,1085 @@
+// ITU-T X.696 (02/2021) version of OER decoding
+// In OER, without knowledge of the type of the value encoded, it is not possible to determine
+// the structure of the encoding. In particular, the end of the encoding cannot be determined from
+// the encoding itself without knowledge of the type being encoded ITU-T X.696 (6.2).
+use super::enc::EncodingRules;
+use crate::oer::ranges;
+use crate::prelude::{
+    Any, BitString, BmpString, Constraints, Constructed, DecodeChoice, Enumerated, GeneralString,
+    GeneralizedTime, Ia5String, Integer, NumericString, ObjectIdentifier, PrintableString, SetOf,
+    TeletexString, UtcTime, VisibleString,
+};
+use crate::types::fields::{Field, Fields};
+use crate::{Codec, Decode, Tag};
+use alloc::{
+    collections::VecDeque,
+    string::{String, ToString},
+    vec::Vec,
+};
+use bitvec::field::BitField;
+use bitvec::macros::internal::funty::Fundamental;
```

Comment:
You should import funty directly instead of using a hidden import.

Reply:
Good catch, seems like `funty` is not even needed and I removed the usage.